### DISCUSSION TITLE
Convert Discussion into Issue

### DISCUSSION THREAD

**Discussion by philsward** [community/community#3133] [Discussions] [answered]
I have not used discussions yet, however I'm looking at it as an alternative to user forums for a variety of projects.

I've asked in another area whether discussions can be converted to issues and the answer was "no".

I feel like the ability to convert a discussion into an issue would be very beneficial. A lot of "discussions" result in issues and it would be nice to see everything that went into the discussion to warrant the issue.

It might be as simple as having a button within discussions that creates a new issue and text at the top of the issue that says: Issue created from discussion [link]

**Comment by evi-liu** [comment, marked as answer]
Thanks for the feedback! This is a duplicate of https://github.com/github/feedback/discussions/2861, so please see that discussion for the latest updates.

**Reply by philsward** [reply]
Thanks, I tried to find one but didn't come up with anything quickly. Sorry for the dupe.

**Reply by becca** [reply]
I'm curious @philsward, how did you go about looking for the Discussion and why wasn't it easy to find? This is also an area we want to improve upon to avoid duplicate discussions so anything you can relay here would be awesome
